lib/rocket_chat/messages/room.rb

```diff
@@ -99,6 +99,44 @@ def remove_owner(room_id: nil, user_id: nil)
       end
 
       #
+      # *.add_moderator REST API
+      # @param [String] room_id Rocket.Chat room id
+      # @param [String] user_id Rocket.Chat user id
+      # @return [Boolean]
+      # @raise [HTTPError, StatusError]
+      #
+      def add_moderator(room_id: nil, user_id: nil)
```

Comment:
Given `room_id` and `user_id` are both required arguments, might be a good idea to validate pre-flight. The server will handle it of course. Thoughts?